hinks.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+"""
+Instructions
+
+- Functions intended to run for each scan
+  need to be named run_xxxxx
+
+- Do not modify the parameters of the run_xxx function
+  unless you know what you are doing
+  see optional parameters:
+  https://github.com/tangible-landscape/grass-tangible-landscape/wiki/Running-analyses-and-developing-workflows#python-workflows
+
+- All gs.run_command/read_command/write_command/parse_command
+  need to be passed env parameter (..., env=env)
+"""
+
+import grass.script as gs
+
```

Comment:
The import statement linting tool _isort_ wants you to include an extra empty line after the imports. (There should be two empty lines between the imports and the function definitions.) The relevant error from the Super-Linter check (which includes the isort tool) is:

```
ERROR: /github/workspace/hinks.py Imports are incorrectly sorted and/or formatted.
--- /github/workspace/hinks.py:before	2021-02-11 15:53:16.684052
+++ /github/workspace/hinks.py:after	2021-02-11 15:54:03.956293
@@ -16,6 +16,7 @@
 """
 
 import grass.script as gs
+
 
 def run_ponds(scanned_elev, env, **kwargs):
     repeat = 20
```